backend/models/volunteer_model.py

```diff
@@ -68,3 +68,12 @@ class Volunteer(Document):
     sent_photo = BooleanField(default=False)
     aggreed_terms = BooleanField(default=False)
     april1 = BooleanField(default=False)
+
+    def clean_data(self) -> dict:
```

Comment:
What is it added for?

Reply:
It was added to return JSON result in existing format
 {
  "count": 3, 
  "list": [
    {
      "_id": "5f712ab2feee02c4a1e7a005" ....

I can remove clean_data method and use something like this to return JSON
`volunteers = [**json.loads(v.to_json())** for v in obj.order_by("-created_at").skip(offset).limit(item_per_age)]
return jsonify({"list": volunteers, "count": obj.count()})`

It will return almost the same JSON structure, but difference will be in next fields:
`"_id": {
        "$oid": "5f712ab2feee02c4a1e7a005"
      }`
`"created_at": {
        "$date": 1601252018190
      }`

If it will not be a problem I'll remove clean_data method from volunteer_model and use **json.loads(v.to_json())**